modules/Sources/Features/SendFlow/SendFlow.swift

```diff
@@ -103,8 +103,7 @@ public struct SendFlow: Reducer {
         public var hasEnteredAmount: Bool { amountToSend > .zero }
         public var hasEnteredRecipient: Bool { recipient?.data.isEmpty != true }
         public var canSendEnteredAmount: Bool {
-            true
-//             amountToSend <= maxAmount
+             amountToSend <= maxAmount
```

Comment:
Uhmmm I think this is not fully accurate in terms of how the Fee is met. I'd manually test the `maxAmount` case to see what happens in reality when the transaction is built and the fee is added.  If you already did that, please disregard this comment